templates/BenchmarkDotNet.Templates.csproj

```diff
@@ -24,4 +24,18 @@
     <Compile Remove="**\*" />
     <None Remove="*.bat;.sh" />
   </ItemGroup>
+
+  <Target Name="Fix BenchmarkDotNetVersion"
+          BeforeTargets="Build">
+    <ReplaceFileText
+            InputFilename="templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json"
```

Comment:
Please add similar tasks for VB and F# templates.

Reply:
fixed at 8ba62e2659b94f6473acfd029138ec22904a5710